test/common.py

```diff
@@ -1228,11 +1227,6 @@ def setUp(self):
         self.cflags += building.get_emcc_node_flags(node_version)
         self.cflags.append('-Wno-transpile')
 
-      # This allows much of the test suite to be run on older versions of node that don't
-      # support wasm bigint integration
-      if node_version[0] < feature_matrix.min_browser_versions[feature_matrix.Feature.JS_BIGINT_INTEGRATION]['node'] / 10000:
-        self.cflags.append('-sWASM_BIGINT=0')
```

Comment:
I'm particularly interested in delete this gross couple of lines.. here we are basically saying that if the node version we are testing against is old enough we change the default settings for `-sWASM_BIGINT`... which seems very hacky.